llarp/win32/abyss.rc

```diff
@@ -89,7 +89,7 @@ BEGIN
 #ifdef LLARP_RELEASE_MOTTO
             VALUE "ProductVersion", VERSION_STRING(0.4.0, RELEASE_MOTTO, GIT_REV)
 #else
-            VALUE "ProductVersion", VERSION_STRING(0.4.0-dev-, GIT_REV)
```

Comment:
hmmm there should be a way to detect when /lib/cpp is being used as opposed to the K&R era RC preprocessor

Reply:
probably. I'm not sure if this is even needed with MSVC, this is what I started digging into.

Reply:
reverted.